.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    # Run at 18:00 UTC every day
+    - cron: '0 18 * * *'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Set 0 to rebase later.
+          fetch-depth: 0
+
+      - name: Read .env
+        run: |
+          . ./.env
+          echo "MDBOOK_VERSION=${MDBOOK_VERSION}" >> $GITHUB_ENV
+          echo "MDBOOK_LINKCHECK_VERSION=${MDBOOK_LINKCHECK_VERSION}" >> $GITHUB_ENV
+          echo "MDBOOK_TOC_VERSION=${MDBOOK_TOC_VERSION}" >> $GITHUB_ENV
+          echo "DEPLOY_DIR=${DEPLOY_DIR}" >> $GITHUB_ENV
```

Comment:
Why did you choose to make this a separate file, rather than putting the versions directly in the CI workflow?

Reply:
Initially, I had two config files and placing values to each file was annoying. Then I forgot to revert that change after cleaning up. Thanks for pointing out :)